### PR TITLE
Bump kubernetes client version and addons version for stress watcher

### PR DESCRIPTION
**Pull request by benbp**
Potentially fixes some issues with the watcher and http connections/re-connections. The latest version of the client has better handling for keepalive issues.

This also fixes build/deploy of the chart, which needed to be updated to the latest addons.

### CODE REVIEW

File: tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs [hunk 1]
```diff
@@ -78,7 +77,9 @@ public async Task Watch(CancellationToken cancellationToken)
             {
                 try
                 {
-                    var listTask = Client.ListPodForAllNamespacesWithHttpMessagesAsync(
+                    Logger.Information("Starting pod watch");
+
+                    var listTask = Client.CoreV1.ListPodForAllNamespacesWithHttpMessagesAsync(
```
Comment (ckairen) [inline tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs:82]: I read a little bit about the CoreV1 and BatchV1 for pod and job, still couldn't figure out the difference between this, and the previous implementation

Comment (ckairen) [reply tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs:82]: other than those 2, lgtm

**Review by ckairen — APPROVED** [review]

File: tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs (hunk 1 above)
Comment (benbp) [reply tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs:82]: @ckairen My understanding is they moved the method organization into classes within their respective API groups, vs. just having them all at the top level like before. A lot of this code is auto-generated from the kubernetes REST spec anyway.

[pod spec, corev1 api group](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/), [job spec, batchv1 api group](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/).

Comment (ckairen) [reply tools/stress-cluster/services/Stress.Watcher/src/PodEventHandler.cs:82]: ahh ok, so it's their API change

**Pull request merged**